heron/simulator/src/java/BUILD

```diff
@@ -8,6 +8,7 @@ simulator_deps_files = \
     heron_java_proto_files() + [
         "//heron/api/src/java:api-java",
         "//heron/common/src/java:common-java",
+        "//heron/instance/src/java:instance-java",
```

Comment:
This leads to the issue that storm-compatibility, (or anything with simulator), contains the whole heron-instance dependencies, which is not expected since it complicates the storm-compatibility's dependencies a lot.